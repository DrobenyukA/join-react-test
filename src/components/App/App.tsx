import React from 'react';
import { ThemeProvider } from 'styled-components';

import AppRouter from './Router';
import { defaultTheme } from '../../styles/themes';

const App = () => {
    const [theme] = React.useState(defaultTheme);

    return (
        <ThemeProvider theme={theme}>
            <AppRouter />
        </ThemeProvider>
    );
};

export default App;
