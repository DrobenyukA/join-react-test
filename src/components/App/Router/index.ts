// istanbul ignore file

export { default } from './AppRouter';
