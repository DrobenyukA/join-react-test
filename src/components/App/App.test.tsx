import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from './App';

describe('<App />', () => {
    let wrapper: ShallowWrapper;

    beforeAll(() => {
        wrapper = shallow(<App />);
    });

    it('Should have correct layout', () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
