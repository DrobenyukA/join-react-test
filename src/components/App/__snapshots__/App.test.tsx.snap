// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Should have correct layout 1`] = `
<div
  className="app"
>
  <AppRouter />
</div>
`;
